packages/manager/modules/hub/src/components/user-panel/payment-mean/payment-method.class.js

```diff
@@ -0,0 +1,25 @@
+export default class PaymentMethod {
+  constructor(paymentMethod) {
+    Object.assign(this, paymentMethod);
+  }
+
+  getStatusCategory() {
```

Comment:
This method may be useful into `ng-ovh-payment-method` no ?

Reply:
yes good point i'm going to refactor it